.travis.yml

```diff
@@ -14,12 +14,15 @@ env:
 
 cache:
   directories:
-    - $HOME/.cache/pip
+    - $HOME/.cache/pypoetry
+
+before_install:
+  - pip install poetry
 
 install:
   - ./scripts/setup.sh
 
 script:
-  - ./scripts/test.sh
-  - ./scripts/benchmark.sh
-  - ./scripts/dist.sh
+  - poetry run ./scripts/test.sh
```

Comment:
Now we're doing virtualenv magic, we shouldn't need the `poetry run` prefix here?

Reply:
c8b01cc